drivers/SmartThings/zwave-lock/src/init.lua

```diff
@@ -68,6 +92,10 @@ local function added_handler(self, device)
   end
 end
 
+local init_handler = function(driver, device, event)
+  populate_state_from_data(device)
```

Comment:
We can discuss if it makes sense to even have this handling in `init` vs just `added` or including `infoChanged`.  I believe that `added` alone should be sufficient, this was just intended as a fallback, so I would be OK with removing it.  I didn't really want to use `infoChanged` because 1) we _shouldn't_ ever receive the data via that path and 2) `infoChanged` happens more frequent than the other lifecycle events and so adding an additional lookup/check seemed like extra work.